static/app/utils/metrics/mri.tsx

```diff
@@ -7,21 +7,28 @@ export const DEFAULT_MRI: MRI = 'c:custom/sentry_metric@none';
 export const DEFAULT_METRIC_ALERT_FIELD = `sum(${DEFAULT_MRI})`;
 
 export function isMRI(mri?: unknown): mri is MRI {
-  return !!parseMRI(mri);
+  try {
+    _parseMRI(mri as string);
```

Comment:
```suggestion
    _parseMRI(String(mri));
```

Reply:
thinking about it, I should simply do a `typeof mri !== string` before calling `_parseMRI`

Reply:
yes that is even better